Home_Task_9.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Please within Pull Request place only files related to hometask. Other files should be deleted.